airflow/gcp/operators/gcs.py

```diff
@@ -526,3 +529,96 @@ def execute(self, context):
                                role=self.role,
                                generation=self.generation,
                                user_project=self.user_project)
+
+
+class GcsFileTransformOperator(BaseOperator):
+    """
+    Copies data from a source GCS location to a temporary location on the
+    local filesystem. Runs a transformation on this file as specified by
+    the transformation script and uploads the output to a destination bucket.
+
+    The locations of the source and the destination files in the local
+    filesystem is provided as an first and second arguments to the
```

Comment:
```suggestion
    filesystem is provided as an X and X arguments to the
```
We don't use ordered parameters. Operators always use a keyword argument.

Reply:
Right and parameters to the operator are passed by keywords arguments. The line talks about passing argument to the script which is run in subprocess.